datadog_checks_dev/datadog_checks/dev/tooling/commands/utils.py

```diff
@@ -0,0 +1,37 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import time
+
+from ...subprocess import run_command
+from .console import abort, echo_failure, echo_success, echo_waiting
+
+
+def run_command_with_retry(retry, command, *args, **kwargs):
+    """ Wrap run_command with retry.
+
+        If retry is None, we just forward the call to run_command.
+        Otherwise, retry run_command until success or retry limit.
+    """
+    if retry is None:
+        return run_command(command, *args, **kwargs)
+
+    if retry < 1:
+        abort('\nRetry must be >= 1.', code=2)
+
+    attempt = 1
+    result = None
+    while attempt <= retry:
+        echo_prefix = "[RETRY] {}/{} - Command \"{}\": ".format(attempt, retry, command)
```

Comment:
Can we please not output on the first run?

Reply:
Sure, skipped all outputs on first retry (except failure)